projects/demo/src/app/app.component.ts

```diff
@@ -71,19 +70,19 @@ export class AppComponent {
         },
     ];
     activeItem = 'icon';
-    _isDesktop = false;
     _isMenuVisible = true;
-    sideNavToggle: Subject<boolean> = new Subject<boolean>();
+    mode?: string;
     constructor(private traverser: Traverser, private pastanaga: PastanagaService, private cdr: ChangeDetectorRef) {
         this.pastanaga.breakpoint.currentMode.subscribe((mode) => {
-            this._isDesktop = mode === 'desktop';
+            this.mode = mode;
+            this._isMenuVisible = mode === 'desktop';
             markForCheck(this.cdr);
         });
         traverser.addView('view', '', WelcomePageComponent);
         this.menu.forEach((section) => section.pages.forEach((page) => traverser.addView(page.view, '', page.type)));
     }
 
-    toggleMenu() {
-        this.sideNavToggle.next(true);
+    toggleSideNav(value: boolean) {
```

Comment:
you're always passing `true` to this method, instead you should rename is `openSideNav` without any argument and just setting `this._isMenuVisible = true;`

Reply:
i'm not always sending true, if you close the side-nav the value will be false.